spring-cloud-modules/spring-cloud-eureka/spring-cloud-eureka-feign-client-integration-test/src/test/java/com/baeldung/spring/cloud/client/TestConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.spring.cloud.client;
+
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
+
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.test.context.ActiveProfiles;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+
+@TestConfiguration
+@ActiveProfiles("test")
+public class TestConfig {
+
+    @Bean(initMethod = "start", destroyMethod = "stop")
+    public WireMockServer mockBooksService() {
+        return new WireMockServer(options().port(80));
+    }
+
+    @Bean(name="secondMockBooksService", initMethod = "start", destroyMethod = "stop")
+    public WireMockServer secondBooksMockService() {
+        return new WireMockServer(options().port(81));
```

Comment:
will this port be available while running in an integration environment in Jenkins.
I think we can use Spring Boot's LocalServerPort for this or something similar, just check

Reply:
@dkapil so even though this PR's default and integration JDK9+ checks are passing green, is it possible to fail after merging ?